doc/SAI_1.7.1 Release notes.md

```diff
@@ -0,0 +1,149 @@
+
+### Add packet allocate, free function to allow for 0 copy packet TX  
+This PR is to add more attribute for querying available packet DMA pool size. This can be used for accounting for and debugging allocations from packet DMA pool.Sending a packet requires allocating it first. Typically, this requires allocating from a special pool of DMAed memory. For this a ASIC vendor SDK may support custom allocate and free functions. So mirror those in the SAI spec.Without such a ability we are left to allocate heap memory, pass it to send_hostif_packet_fn, which then allocates from the afore mentioned DMA memory, copies the contents of heap memory and sends the packet along. This causes us to have a extra allocation and copy.
+This is achieved by adding SAI_HOSTIF_PACKET_ATTR_ZERO_COPY_TX in sai_hostif_packet_attr_t and adding a type def sai_allocate_hostif_packet_fn for saihostif.h and adding SAI_SWITCH_ATTR_PACKET_DMA_MEMORY_POOL_SIZE in sai_switch_attr_t for saiswitch.h
+
+The PR related to this feature is available at [PR#1137](https://github.com/opencomputeproject/SAI/pull/1137)
+
+### Add label attribute for LAG and virtual router 
+Since LAG and virtual router can be created without any unique mandatory attributes, anchor attribute can be used to uniquely identify specified objects. This will be very useful in SONiC warm boot scenario, for example to identify 2 empty LAGs which are present on the device after reboot.
+This attribute doesn't correspond to any internal SAI vendor device resources, and at this point it don't need to be implemented by any vendor, it implementation at create function internals CAN be skipped without posting error. This attribute is considered as user data attached to specific object.
+
+The PR related to this feature is available at [PR#1158](https://github.com/opencomputeproject/SAI/pull/1158)
+
+### Add neighbor and tunnel term ip addr family for CRM 
+Generic resource monitoring, done by API sai_object_type_get_availability, introduced the ability to query avaialble resources per object type, and provide additional attributes as query params. This introduces 2 such params - neighbor entry IP address family, to be able to query avaialble IPv4 and IPv6 neigbor entries, and tunnel termination table entry IP address family, to be able to query available IPv4 and IPv6 tunnel termination table entries. Introduced attributes are read only, marked as resourcetype, and used only for this query by adding attribute SAI_NEIGHBOR_ENTRY_ATTR_IP_ADDR_FAMILY in enum sai_neighbor_entry_attr_t for saineighbor.h and attribute SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_IP_ADDR_FAMILY in enum sai_tunnel_term_table_entry_attr_t for saitunnel.h
+
+The PR related to this feature is available at [PR#1151](https://github.com/opencomputeproject/SAI/pull/1151)
+
+### UDF Field Qualifier in ACL 	
+This feature brings in change to be able to specify field extracted by UDF group as a qualifier in ACL Table for typedef enum _sai_acl_table_attr_t 
+

```

Comment:
Please add following line:
A UDF group that will be used in ACL must be set to SAI_UDF_GROUP_GENERIC as the group type.

Reply:
addressed